ftplugin/go/commands.vim

```diff
@@ -85,6 +85,12 @@ if globpath(&rtp, 'plugin/ctrlp.vim') != ""
   command! -nargs=? -complete=dir GoDeclsDir call ctrlp#init(ctrlp#decls#cmd(1, <q-args>))
 endif
 
+" -- fzf
+if globpath(&rtp, 'plugin/fzf.vim') != ""
+  command! -nargs=? -complete=file GoDecls call fzf#decls#cmd(0, <q-args>)
+  command! -nargs=? -complete=dir GoDeclsDir call fzf#decls#cmd(1, <q-args>)
+endif
```

Comment:
@fatih can you point to somewhere in the codebase where a similar option was implemented. I'd love to see support for fzf in vim-go and am willing to contribute. I have little experience with vim-scripting so having an example would help.